### PR TITLE
Release 0.1.0

### PR DESCRIPTION
**Pull request by justinsb**
```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,7 +22,7 @@ export KUBEBUILDER_CONTROLPLANE_START_TIMEOUT ?=60s
 export KUBEBUILDER_CONTROLPLANE_STOP_TIMEOUT ?=60s
 
 # Image URL to use all building/pushing image targets
-export CONTROLLER_IMG ?= gcr.io/k8s-cluster-api/cluster-api-controller:latest
+export CONTROLLER_IMG ?= gcr.io/k8s-cluster-api/cluster-api-controller:0.1.0
 export EXAMPLE_PROVIDER_IMG ?= gcr.io/k8s-cluster-api/example-provider-controller:latest
```
Comment (justinsb) [inline Makefile]: Note that I am _not_ changing the EXAMPLE_PROVIDER_IMG ... I figure that's for CI and we should not release it (?)

Comment (vincepri) [reply Makefile]: I think that correct

/cc @figo

**Comment by justinsb** [comment]
Starting to document the deltas from the pre-numbered release process in #862

File: Makefile (hunk 1 above)
Comment (figo) [reply Makefile]: CI will set the EXAMPLE_PROVIDER_IMG based on git hash at runtime. so the tag `latest` here does not matter, hence no need to change.

Comment (detiber) [reply Makefile]: Should we prepend the version with a v (note I'm not overly opinionated either way)?

**Pull request merged**